### DISCUSSION TITLE
'Find' behaviour changed?

### DISCUSSION THREAD

**Discussion by GrahamBodley** [freeplane/freeplane#1953] [Help] [answered]
Maybe you can already do what I need, but if so, I cannot find how! Can anyone advise me please?

The behaviour of 'Finding' mindmap entries has changed, and I find the current versions are less useful.

With Ctrl-G Find, it will find the nodes, even if they have been collapsed and are not visible; the Find window stays open, and you can 'Find previous' and 'Find next', but you cannot leave the modal Ctrl-G 'Find' windown until you press 'OK'; but with that Ctrl-G window open, you cannot pan the map (to get a better view, see adjacent entries to help confirm 'yes, this is the node I am searching for'. So, you need to keep switching windows, until you find the one you want; previously, you didn't.

With a Ctrl-F Find, you can move around in the map immediately without problem, and click 'Find Next **Visible**' or 'Find Previous **Visible**', so no need to keep switching windows.
The behaviour used to be, it would search inside collapsed (invisible) nodes too; a Ctrl-F Find would expand the Found branch so you could see it in context, then a 'Find Next' would restore the previous 'state of collapse' before finding the next match.

But the behavior now is 'Find next **visible**' and I can't see how to disable the '_**only if visible**_' filter.

Would it be possible to add a '**_Behaviour Option_**' to choose between 'Search **ALL**' and 'Search **VISIBLE**' with Ctrl-F (which I find more friendly than Ctrl-G)?
Or can you do this alread?
I am using version 1.11.14

Mant thanks.
GrahamB

**Comment by dpolivaev** [comment, marked as answer]
<img width="450" alt="find" src="https://github.com/user-attachments/assets/cd4b66a7-6f54-43cb-b61d-3dbc6682269f">

<img width="270" alt="findMenu" src="https://github.com/user-attachments/assets/41edf72c-3364-49c7-8ded-3c4acb9dd19b">

Why are these options not sufficient?

**Reply by dpolivaev** [reply]
You can assign menu accelerators to any of them

**Reply by GrahamBodley** [reply]
Thank you so much for your speedy reply; I am clearly in need of a brain transplant!

**Reply by dpolivaev** [reply]
I am sorry I do not have any extra brain
